internal/datasourcev2/streaming_data_source.go

```diff
@@ -304,11 +304,8 @@ func (sp *StreamProcessor) consumeStream(stream *es.Stream, closeWhenReady chan<
 	}
 }
 
-func (sp *StreamProcessor) subscribe(closeWhenReady chan<- struct{}, selector fdv2proto.Selector) {
+func (sp *StreamProcessor) subscribe(closeWhenReady chan<- struct{}, _ fdv2proto.Selector) {
```

Comment:
Do you think this param will ever be used agian?

Reply:
https://github.com/launchdarkly/go-server-sdk/pull/228